### PR TITLE
Fix format error in "copyable" codeblocks and other type error

### PR DESCRIPTION
**Pull request by handlerww**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/advanced-statefulset.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The [advanced StatefulSet controller](https://github.com/pingcap/advanced-statef
 
 2. Enable the `AdvancedStatefulSet` feature in `values.yaml` of the TiDB Operator chart:
 
-    {{< copyable "shell-regular" >}}
+    {{< copyable "yaml" >}}
```
Comment (ran-huang) [inline en/advanced-statefulset.md:39]: For `yaml`, please use `{{< copyable "" >}}`. Thanks!

Comment (handlerww) [reply en/advanced-statefulset.md:39]: ok

**Review by ran-huang — APPROVED** [review]
Thank you!

**Pull request merged**